frontend-web/webclient/app/Applications/Jobs/View.tsx

```diff
@@ -1324,49 +1451,11 @@ function getAppType(job: Job): string {
     return job.status.resolvedApplication!.invocation.applicationType;
 }
 
-function useRedirectToWebLinks(job?: Job): string[] {
-    const [links, setLinks] = useState<string[]>([]);
-    const [, call] = useCloudCommand();
-
-    React.useEffect(() => {
-        if (!job) return;
-        const appType = getAppType(job);
-        const backendType = getBackend(job);
-        const support = job.status.resolvedSupport ?
-            (job.status.resolvedSupport! as ResolvedSupport<never, ComputeSupport>).support : undefined;
-        const supportsInterface =
-            (appType === "WEB" && isSupported(backendType, support, "web")) ||
-            (appType === "VNC" && isSupported(backendType, support, "vnc"));
-
-        if (appType === "WEB" && supportsInterface && job.status.state === "RUNNING") {
-
-            const requests: OpenInteractiveSessionRequest[] = Array(job.specification.replicas)
-                .fill(0).map((_, i) => ({sessionType: "WEB", id: job.id, rank: i}));
-
-            (async () => {
-                try {
-                    const result = await call<BulkResponse<InteractiveSession>>(JobsApi.openInteractiveSession(bulkRequestOf(...requests)));
-                    setLinks(links => {
-                        for (const res of result?.responses ?? []) {
-                            links[res.session.rank] = (res.session as WebSession).redirectClientTo
-                        }
-                        return links;
-                    })
```

Comment:
wrt. what we talked about yesterday @xiroV, `setLinks` from the `useState` above is being used here after `results` are fetched. This is why this works, as it forces a re-render.